doc/reference/configuration/configuration_reference.rst

```diff
@@ -3,13 +3,280 @@
 Configuration reference
 =======================
 
+..  TODO
+    https://github.com/tarantool/doc/issues/3664
+
 This topic describes all :ref:`configuration parameters <configuration>` provided by Tarantool.
 
+
+..  _configuration_reference_config_parameter:
+
+<config_parameter>
+------------------
+
+Any configuration parameter that can be defined in the global :ref:`scope <configuration_scopes>`.
+For example, :ref:`iproto <configuration_reference_iproto>` and :ref:`database <configuration_reference_database>` configuration parameters defined at the global level are applied to all instances in the cluster.
```

Comment:
AFAIU, it is just a placeholder, a kind of 'any parameter, but see scope constraints for certain parameters'.

It feels confusing that we highliht the global scope here. Can we consider some alternative wordings?

Reply:
Renamed the `<config_parameter>` section to `Global scope` and rephrased a bit.

Reply:
> Most of the configuration options described in this reference can be applied to a specific instance, replica set, group, or to all instances globally. To do so, you need to define the required option at the [global level].

It still looks like we highlight the global options specifically. Can we, for example, say the following (global -> appropriate)?

> To do so, you need to define the required option at the [appropriate level].

We also highlight the global scope second time in the same paragraph.

> For example, [iproto] and [database] configuration parameters defined at the global level are applied to all instances in the cluster.

I think that it is redundant.

Reply:
> It still looks like we highlight the global options specifically. Can we, for example, say the following (global -> appropriate)?

Sorry, looks like I fixed it right after you started reviewing it, now it looks as follows:
```
To do so, you need to define the required option at the :ref:`specified level <configuration_scopes>`.
For example, :ref:`iproto <configuration_reference_iproto>` and :ref:`database <configuration_reference_database>` configuration parameters defined at the global level are applied to all instances in the cluster.
```

> For example, [iproto] and [database] configuration parameters defined at the global level are applied to all instances in the cluster.

Removed